docs/docs/forms.md

```diff
@@ -383,6 +383,67 @@ const RequiredField = ({ label, name, validation }) => {
 }
 ```
 
+### Controlled Component Fields
+
+If you're working with a fully-loaded component library, or bring your own production-ready components you may want them to integrate with Redwood seemlessly.
+
+You can create these custom fields through the use of Redwood's `useErrorStyles` and React Hook Form's `Controller`. 
+
+The following example shows a component from [`primereact`](https://www.primefaces.org/primereact/) which can be used in Forms like any of the named input fields (listed above).
+
+```jsx
+import { ToggleButton, ToggleButtonProps } from 'primereact/togglebutton'
```

Comment:
Just a personal preference:
```suggestion
import { ToggleButton } from 'primereact/togglebutton'
import type { ToggleButtonProps } from 'primereact/togglebutton'
```

Reply:
Another personal preference, just helps upfront for readability to say it's a type